automation/run-tests.sh

```diff
@@ -203,6 +203,11 @@ function upgrade_nm_from_copr {
     local copr_repo=$1
     # The repoid for a Copr repo is the name with the slash replaces by a colon
     local copr_repo_id="copr:copr.fedorainfracloud.org:${copr_repo/\//:}"
+    # Workaroudn for dnf failure:
```

Comment:
Should this maybe be part of the container spec/Dockerfile? Also there is a typo in "Workaround".

Reply:
This only happens in travis.

I tried in my VM with ubuntu 18.04, no such failure.